### ISSUE TITLE
Question about ablation study

### ISSUE THREAD

**Issue by happywinder** [PKU-ICST-MIPL/Finedefics_ICLR2025#7]
Q:Why are the results of fine-tuning only on the open set compared with the results of fine-tuning on both the open and closed sets in the main text, while the metric is the evaluation on the closed set?

**Comment by hehulingxiao** [comment]
Thanks for your thoughtful question. In the main text, finetuning on the open set is served as a naive baseline for comparison, as the original FGVR data format is (image-label) pair instead of (image-candidates-label) triple. We follow FOCI benchmark to evaluate on the closed set, as it is still difficult to find a robust metric to measure the results on the open set.
